ember-cli-build.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (defaults) {
 
   const { maybeEmbroider } = require('@embroider/test-setup');
   return maybeEmbroider(app, {
+    staticHelpers: false,
```

Comment:
This technique fundamentally does not work with Embroider strict mode. (Specifically the `staticHelpers` config + ambiguous attribute position). We could resolve this if we had access to embroider's static resolution.